### PR TITLE
Update Filters 4

### PR DESCRIPTION
**Pull request by wout-codes**
1. `https://egydead.com/`
2. `http://9putlocker.io/watch/OGgYlw1v-the-intruder-2019.html`
3. `https://5movies.cloud/film/crown-and-anchor-31934/watching.html?ep=31934_0_1`
4. `movies2.com.pk`
5. `https://www.vidbm.com/embed-v0ferq1c7x8q.html`
6. `newmusicalbum.host`
7. `r2sa.net`
8. `http://uppoom.com/ecsewdpht8ie/STKOSKMHD___40_2020__41__www.SkymoviesHD.ltd_720p_HDRip_HQ_Dual_Audio_x264_AAC.mkv.html`
9. `https://uploadmoon.com/d0zza24u4v51`
10. #78571
11. `http://imgviv.xyz/2dcsxlf`
12. `speedsatoshi.com` (requires an account - BAB Detection)
13. `buffstreams.tv`
14. `live-sport24.com`
15. `https://topcryptoz.net/?link=8ANRNU`
16. `http://streamsport.pro/live/s3.php`

### CODE REVIEW

**Comment by wout-codes** [comment]
@AdamWr I would be very thankful to you, if you would review this PR. Thank you.

**Comment by AdamWr** [comment]
Too many websites and conflicts, I will check these websites one by one and add rules directly.
Fixed:
```
egydead.com
9putlocker.io
```

**Comment by wout-codes** [comment]
I still can reproduce. I hope that you will merge this anyhow.

**Comment by Alex-302** [comment]
It can't be merged because of a lot of conflicts. This is a reason why we ask to do not fix more than one site(similar can be accepted) in one PR.

**Comment by wout-codes** [comment]
Wait

**Comment by wout-codes** [comment]
@AdamWr You made things more complicated. I wish I haven't pinged you.

**Comment by AdamWr** [comment]
Only these websites need to be fixed yet:
```
speedsatoshi.com (requires an account - BAB Detection)
buffstreams.tv
live-sport24.com
https://topcryptoz.net/?link=8ANRNU
http://streamsport.pro/live/s3.php
```
and maybe this `5movies.cloud` but I can't reproduce popups on this website.

I will check them later.

**Comment by wout-codes** [comment]
No, no. Don't check anything, everything will be more complicated. I will Fix things up. Watch and learn ;)

**Comment by AdamWr** [comment]
So please make one pull request per one website and there will be no problem.

**Comment by wout-codes** [comment]
I don't believe in impossible. :)

**Comment by wout-codes** [comment]
> So please make one pull request per one website and there will be no problem.

So if there are 30 domains, I should create 30 pull requests? I see, nice way to level up in the total amount of commits.

**Comment by AdamWr** [comment]
>So if there are 30 domains, I should create 30 pull requests?

Yes.

**Comment by wout-codes** [comment]
@AdamWr @Alex-302 Please check before any other complex "Resolve Conflicts" arrive.

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -61,8 +61,6 @@ nevasport.ru#$#.main-wrapper { margin-top: 0 !important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/66030
 naver.com#$#body #NM_INT_RIGHT > div.column_fix_wrap { position: relative!important; }
 naver.com#$#body #NM_INT_RIGHT > div.column_fix_wrap { top: 0!important; }
-! https://github.com/AdguardTeam/AdguardFilters/issues/65375
-web.cnvids.com#$#.InnerDownload > #downloadbtn { display: block !important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/tweaks.txt:65]: why removed? Not mentioned in description

**Review by Alex-302 — APPROVED** [review]
seems to be ok

**Review by AdamWr — APPROVED** [review]

**Pull request merged**